File: tests/fixtures/java/NestedControlFlow.java
// Expected metrics:
//   cyclomatic_complexity: TBD
//   cognitive_complexity: TBD
//   lines_of_code: TBD
//   nesting_depth: TBD

class NestedControlFlow {
    int process(int[] items) {
        int total = 0;
        if (items.length > 0) {
            for (int i = 0; i < items.length; i++) {
                if (items[i] > 0) {
                    total += items[i];
                }
            }
        }
        return total;
    }
}

File: tests/fixtures/php/nested_control_flow.php
<?php
// Expected metrics:
// process: cognitive=TBD, cyclomatic=TBD, sloc=TBD
function process($items) {
    $total = 0;
    if (count($items) > 0) {
        for ($i = 0; $i < count($items); $i++) {
            if ($items[$i] > 0) {
                $total += $items[$i];
            }
        }
    }
    return $total;
}
